releasenotes/notes/bump-check-runners-4-4ec46f9d8a2cae3e.yaml

```diff
@@ -0,0 +1,21 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+prelude:
```

Comment:
The prelude adds it at the top of the release notes. Given the sensitivity of the change I figured it might make sense to put it here, but I'm not sure.